app/build.gradle

```diff
@@ -75,25 +75,25 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
     implementation 'androidx.core:core-ktx:1.9.0'
 
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.6.0'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.6.1'
     implementation 'androidx.navigation:navigation-compose:2.5.3'
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4'
 
     // CameraX
-    implementation 'androidx.camera:camera-camera2:1.2.1'
-    implementation 'androidx.camera:camera-lifecycle:1.2.1'
-    implementation 'androidx.camera:camera-view:1.2.1'
+    implementation 'androidx.camera:camera-camera2:1.2.2'
+    implementation 'androidx.camera:camera-lifecycle:1.2.2'
+    implementation 'androidx.camera:camera-view:1.2.2'
 
```

Comment:
```suggestion
```

These are old dependencies of my own Pull request and should definitely removed. There may be old import that still uses them but they shouldn't be any trouble dealing with them in local. Anyway sorry for the mistake

Reply:
These dependencies made it to the `master` branch, from where I pulled...

Definitely good to know, I'll remove them 👍

Reply:
Thanks and again, sorry for the mistake'